lib/livebook_web/live/session_live.ex

```diff
@@ -1188,6 +1191,21 @@ defmodule LivebookWeb.SessionLive do
     {:noreply, socket}
   end
 
+  def handle_info({:env_vars_changed, env_vars}, socket) do
+    for env_var <- socket.assigns.env_vars do
+      System.delete_env(env_var.key)
+    end
```

Comment:
I'm not so sure if it's a good idea, but if the old env var is deleted from Ets, should we delete it from runtime or keep as is?